AWGL/somatic_db
# Fix Gnomad bugs in issues 77 & 78

## Change description
- Gnomad score displays as a decimal, turn into a percentage (#77)
- Gnomad link for build 37 is incorrect (#78) 
- Added tests for all gnomad functions
- Updated CRM variants file so that it has Gnomad data in it

## Justification for minor update
No effect on processing of data, only how it's visualised

## Testing description
- Check that Gnomad score rounding is correct
- Check that formatting of Gnomad score is correct (percentage not a decimal, to 2dp)
- Check that Gnomad links go to correct variant and genome build (Note - Gnomad was recently updated and only shows v3.1 on the website, whereas our build 38 variants are annotated with v3.0, so there are some very small differences in the values, this has been captured in https://github.com/AWGL/somatic_enrichment_nextflow/issues/27)

## Test data location
- Existing SVD test data

## Verification results
- All manual testing and unit tests passed
- Deployment plan: Simple git pull, no need to make any migrations
